Web Inspector: Autocomplete for text-indent property suggests -webkit-each-line and -webkit-hanging incorrectly

Ahmad-S792 opened the pull request:
#### b24a216b175d9bcf1e318e013acfe1e2c357b498
<pre>
Web Inspector: Autocomplete for text-indent property suggests -webkit-each-line and -webkit-hanging incorrectly

<a href="https://bugs.webkit.org/show_bug.cgi?id=269307">https://bugs.webkit.org/show_bug.cgi?id=269307</a>

Reviewed by Timothy Hatcher.

This PR is to update `autocompletion` list to not suggest prefixed
properties of `hanging` and `each-line` for `text-ident`.

Both of these properties are enabled since 235896@main.

* Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js:
(text-ident):

Canonical link: <a href="https://commits.webkit.org/274979@main">https://commits.webkit.org/274979@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/56381e6ce11dcc0b9655a3207b2f04c1c52926ea

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/39557 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/18536 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/41911 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/42091 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/35457 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/21420 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/15865 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/33038 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/40131 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/15617 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/34240 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/13558 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/13533 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/35191 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/43369 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/35908 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/35522 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/39317 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/14369 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/11835 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/37578 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/15975 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9088 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/16023 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/15632 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

rcaliman-apple reviewed (approved):
Looks good to me 👍 Thanks for updating this! 
@dcrousso or @patrickangle to approve so it can go through merge queue.

Ahmad-S792 committed:
Web Inspector: Autocomplete for text-indent property suggests -webkit-each-line and -webkit-hanging incorrectly

https://bugs.webkit.org/show_bug.cgi?id=269307

Reviewed by Timothy Hatcher.

This PR is to update `autocompletion` list to not suggest prefixed
properties of `hanging` and `each-line` for `text-ident`.

Both of these properties are enabled since 235896@main.

* Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js:
(text-ident):

Canonical link: https://commits.webkit.org/274979@main
